### PR TITLE
Add index count to index info

### PR DESCRIPTION
**Pull request by gvolfing**
closes: https://github.com/memgraph/memgraph/issues/1207

### CODE REVIEW

File: src/query/interpreter.cpp [hunk 1]
```diff
@@ -2910,18 +2910,21 @@ PreparedQuery PrepareInfoQuery(ParsedQuery parsed_query, bool in_explicit_transa
...
       handler = [interpreter_context] {
         auto *db = interpreter_context->db.get();
         auto info = db->ListAllIndices();
         std::vector<std::vector<TypedValue>> results;
         results.reserve(info.label.size() + info.label_property.size());
+        auto acc = db->Access();
         for (const auto &item : info.label) {
-          results.push_back({TypedValue("label"), TypedValue(db->LabelToName(item)), TypedValue()});
+          results.push_back({TypedValue("label"), TypedValue(db->LabelToName(item)), TypedValue(),
+                             TypedValue(static_cast<int>(acc->ApproximateVertexCount(item)))});
```
Comment (antoniofilipovic) [inline src/query/interpreter.cpp:2922]: How correct is this number, by how much it can miss?

**Comment by antoniofilipovic** [comment]
It looks good to me but I just want to check on this approximation how correct it is

**Review by Josipmrden — APPROVED** [review]

File: src/query/interpreter.cpp (hunk 1 above)
Comment (gitbuda) [reply src/query/interpreter.cpp:2922]: It depends, it's the number of non-deleted elements inside the skiplist, for the sake of analysis from the users perspective should be accurate enough -> resolving this for now 😃

**Review by gitbuda — CHANGES_REQUESTED** [review]
`e2e/replication` fails on this PR (after resolving the merge conflict, not sure if it has passed before)
```
cd tests/e2e
./run.sh "Show while creating invalid state"
```
Screenshot 2023-10-23 at 10 02 30 PM

File: tests/e2e/replication/show_while_creating_invalid_state.py [hunk 2]
```diff
@@ -1052,7 +1052,7 @@ def retrieve_data():
     # 5/
     expected_data = {
         ("sync_replica1", "127.0.0.1:10001", "sync", 0, 0, "invalid"),
-        ("sync_replica2", "127.0.0.1:10002", "sync", 5, 0, "ready"),
+        ("sync_replica2", "127.0.0.1:10002", "sync", 6, 0, "ready"),
```
Comment (gitbuda) [inline tests/e2e/replication/show_while_creating_invalid_state.py:1055]: @gvolfing what does that number represents?

Comment (gvolfing) [reply tests/e2e/replication/show_while_creating_invalid_state.py:1055]: `current_timestamp_of_replica`

Comment (gvolfing) [reply tests/e2e/replication/show_while_creating_invalid_state.py:1055]: It has been nugded a couple of times in these past days.

Comment (gitbuda) [reply tests/e2e/replication/show_while_creating_invalid_state.py:1055]: why exactly did that change here? 🤔

**Review by gitbuda — APPROVED** [review]

File: tests/e2e/replication/show_while_creating_invalid_state.py (hunk 2 above)
Comment (gvolfing) [reply tests/e2e/replication/show_while_creating_invalid_state.py:1055]: To call `ApproximateVertexCount` we need a storage accessor, which will prop up the count.

**Pull request merged**